test/WebJobs.Script.Tests.Integration/ContainerManagement/LinuxContainerInitializationHostedServiceTests.cs

```diff
@@ -22,14 +23,14 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.ContainerManagement
 {
     [Trait(TestTraits.Category, TestTraits.EndToEnd)]
     [Trait(TestTraits.Group, TestTraits.ContainerInstanceTests)]
-    public class LinuxContainerInitializationHostServiceTests : IDisposable
+    public class LinuxContainerInitializationHostedServiceTests : IDisposable
```

Comment:
should this be AtlasContainerInitializationHostedServiceTests ?

Reply:
Changed!